server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileLookup.java

```diff
@@ -41,47 +38,38 @@ public QProfileLookup(DbClient db) {
     this.db = db;
   }
 
-  public List<QProfile> allProfiles(DbSession dbSession, OrganizationDto organization) {
-    return toQProfiles(db.qualityProfileDao().selectAll(dbSession, organization), organization);
+  public List<QualityProfileDto> allProfiles(DbSession dbSession, OrganizationDto organization) {
+    return db.qualityProfileDao().selectAll(dbSession, organization);
   }
 
-  public List<QProfile> profiles(DbSession dbSession, String language, OrganizationDto organization) {
-    return toQProfiles(db.qualityProfileDao().selectByLanguage(dbSession, language), organization);
+  public List<QualityProfileDto> profiles(DbSession dbSession, String language, OrganizationDto organization) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "organization". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)